.gitignore

```diff
@@ -0,0 +1,21 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
```

Comment:
Remove this comment as its redundant here

Reply:
Thanks for this, Comments deleted